test/promql/tests/test_notready_pods.yaml

```diff
@@ -32,8 +32,12 @@ tests:
               phase: Pending
               source_cluster: cluster01
             exp_annotations:
-              summary: "Pod pod-1 is not ready"
-              description: "Pod pod-1 in namespace ns-1 on cluster cluster01 is not ready for more than 15 minutes."
+              summary: >-
+                Pod pod-1 is not ready
+              description: >-
+                Pod pod-1 in namespace ns-1 on cluster cluster01 is 
```

Comment:
@Kousalya1998 yaml-lint is failing at this line with trailing space error. Please remove extra space at the end. Everything else looks good.

Reply:
I updated. Guess /retest was required to update that page.